.circleci/config.yml

```diff
@@ -519,12 +519,38 @@ jobs:
             if git diff --name-only main HEAD | grep -E 'schemas/'
               then
                 echo "Changes detected in /schemas directory. Running Schemas tests and linting."
-                env
                 make schemas_check
               else
                 echo "No changes in /schemas directory. Skipping tests and linting."
                 circleci-agent step halt
             fi
+  schemas_deploy:
+    docker:
+      - image: cimg/python:3.10.8 # poetry 1.2.2
+    steps:
+      - checkout
+      - run:
+          name: Check for changes in schemas/ in last commit before proceeding.
+          command: |
+            if git diff --name-only main HEAD~1 | grep -E 'schemas/'
+              then
+                echo "Found changes to schemas dir, proceeding with deployment."
+              else
+                echo "No changes in /schemas directory. Skipping mozilla-nimbus-schemas deployment."
+                circleci-agent step halt
+            fi
```

Comment:
This will deploy after every schema change? I think we should only deploy if the toml file changes, because we would potentially deploy a change without updating the version.

Reply:
Yea, that's a good idea. I was a little worried about that scenario, but didn't think to just check for an update in the `pyproject.toml`.

Reply:
@jrbenny35 Yes totally.  I think we only want to deploy on version bumps.  I've been confused about how to deal with that in this repo since using repo tags might not make sense.  But checking on version bumps is a great idea.  And yeah exactly @mikewilli that's a good idea since the pyproject can change for unrelated reasons.  Let's try all that good thinking both 👍

Reply:
I am regretting my suggestion. With the old `setup.py` project configuration, it was easy to use python to dynamically parse a `VERSION` file, but with TOML that isn't possible. I went with what seemed like the most sensible approach that preserves the current poetry setup and the requirement of a single version definition in `pyproject.toml`: I changed the git diff check to look only in the `schemas/pyproject.toml` path, and then grep for `version` in the changes. It's still possible that this could be tricked, but to me this seems good enough. Open to suggestions though.